### ISSUE TITLE
Slider gone from latest version?

### ISSUE THREAD

**Issue by theRealPadster** [czarny/Brisync#33]
I just updated to 1.3 and the slider in the notification panel is gone. Is the equalizer-curve sort of this supposed to replace it? It doesn't seem to do anything when I change it. I used to use the slider as a quick way to adjust my monitor brightness without having to fiddle with my monitor's buttons and OSD. 

EDIT: Hmm, it looks like if I adjust the EQ slider for the brightness level I'm currently on, it does change, but then when I close the settings dialog, it no longer changes when I change my laptop brightness. The only way I can get it to change is by opening the dialog and changing the current EQ slider manually.

**Comment by czarny** [comment]
EQ sliders are supposed to configure curve which is used to match brightness of build-in and external display. The idea is that when curve is ready you don't need to adjust brightness anymore manually.

**Comment by theRealPadster** [comment]
That makes sense, it just doesn't seem to be working for me. Is some sort of delay expected, or is there anything I can to to troubleshoot?

**Comment by czarny** [comment]
Max 1 second of delay is possible. Please make sure all settings panels are closed. If you see brightness changes when you change value of slider corresponding to brightness level of build-in display all should work.

PS. Just released new version which fixes max brightness value sent to display. Maybe it will solve your problems.

**Comment by theRealPadster** [comment]
Ah, something must have gotten messed up because when I quit and restarted it, it was working. It seems to kinda get stuck at the max brightness sometimes or something, but appears to be working now. Is that what you just fixed? I'm using it through homebrew so hopefully that gets updated soon.

**Comment by theRealPadster** [comment]
@czarny Just got the update, and it seems fairly reliable, except it still gets stuck at full brightness when I increase the brightness to max.

**Comment by czarny** [comment]
Any chance to connect on skype / teamviewer so I can check what happens in code?

**Comment by theRealPadster** [comment]
Yeah sure, we can do TeamViewer some time. What timezone/time works for you? I'm in EST and could do tomorrow evening or on the weekend some time.

**Comment by czarny** [comment]
I am in CET, what you say to set a meeting at 18:00 EST?

**Comment by theRealPadster** [comment]
Sure. How does teamviewer work again, is there voice or chat we can use?

**Comment by czarny** [comment]
It is possible to chat for sure. What day do you prefer?

**Comment by theRealPadster** [comment]
Ah, perfect. Today or Tuesday should work.

**Comment by czarny** [comment]
Today is fine for me, 18:00 EST please contact me on gmail, thx.

**Comment by sumerman** [comment]
Apologies for jumping into your discussion. I've discovered another use case where the slider could be handy: when I close my laptop's lid, keyboard shortcuts stop working. The slider allowed me to control the external display brightness even with the lid closed in the previous version.
